automation/default_browser_params.json

```diff
@@ -7,6 +7,8 @@
     "cookie_instrument": false,
     "js_instrument": false,
     "cp_instrument": false,
+    "http_instrument": false,
```

Comment:
Yes, once I merge this I'm going to push a new commit that changes all of the defaults to the extension (including all tests).